### ISSUE TITLE
AWS cli installation on RHEL 7 arm 64 failling

### ISSUE THREAD

**Issue by pgouda89** [aws/aws-cli#6758]
**Describe the bug**
AWS cli installation on RHEL 7 arm 64 is failing with the following error:
/aws/dist/cryptography/hazmat/bindings/_rust.abi3.so: symbol __cxa_thread_atexit_impl, version GLIBC_2.18 not defined in file libc.so.6 with link time reference

Instructions followed to install aws cli: https://aws.amazon.com/blogs/developer/aws-cli-v2-now-available-for-linux-arm/

**SDK version number**
I believe it is the latest aws cli version as we are downloading 'https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip'
**Platform/OS/Hardware/Device**
RHEL 7.6 Arm64
Image: RHEL-7.6_HVM_GA-20181122-arm64-0-Hourly2-GP2

**To Reproduce (observed behavior)**
Create an AWS instance using RHEL-7.6_HVM_GA-20181122-arm64-0-Hourly2-GP2 image
Run following commands in sequence 
curl -O 'https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip'
 unzip awscli-exe-linux-aarch64.zip
sudo ./aws/install
**Expected behavior**
AWS cli installation should go fine(It was working fine few weeks back, I am sure it is a regression)
**Logs/output**
/usr/aws-cli/v2/dist/cryptography/hazmat/bindings/_rust.abi3.so: symbol __cxa_thread_atexit_impl, version GLIBC_2.18 not defined in file libc.so.6 with link time reference

**Comment by pgouda89** [comment]
Hi @tim-finnigan,
Thanks for responding, I am using Python 2.7.5 on RHEL 7.6 machine.
Essentially I am trying the following command sequence to install aws cli:
$ curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip"
unzip awscliv2.zip
sudo ./aws/install

**Comment by tim-finnigan** [comment]
Hi @pgouda89 thanks for following up. We announced the end of Python 2.7 support a while back: https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-python-2-7-in-aws-sdk-for-python-and-aws-cli-v1/

I’m not sure if that or other incompatibilities with your setup are causing issues. We haven’t seen anyone else reporting this issue but will continue to monitor.

**Comment by MarcinZukowski** [comment]
Seeing the exact same issue on Centos7 on aarch64.

I checked older versions, and the latest working one is `2.4.20`. From `2.4.21` this problem occurs.

The latest (`2.4.23`) x86 awscli works for me on an equivalent x86 box, interestingly, the `_rust.abi3.so` file is not present there

**Comment by tim-finnigan** [comment]
I was able to install ARM v2.4.21 on RHEL 8.6 without any issue. Does anyone have updates as far as what they have tried? Still haven’t seen this reported elsewhere.

**Comment by MarcinZukowski** [comment]
Here's a simple Dockerfile that showcases the problem on aarch64 Centos7. 
Again, 2.4.20 works, 2.4.21 doesn't. 
Also, RHEL8 works.

```
FROM centos:7   # doesn't work
# FROM registry.access.redhat.com/ubi8/ubi:8.1   # works

WORKDIR /tmp

RUN yum -y install unzip
RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64-2.4.21.zip" -o "awscliv2.zip"
RUN unzip awscliv2.zip
RUN cd aws &&  ./install -u -i /tmp/aws-cli -b /tmp/aws-cli-bin
RUN /tmp/aws-cli-bin/aws --version
```

**Comment by tim-finnigan** [comment]
Thanks @MarcinZukowski I was able to reproduce the issue with that Dockerfile. The team is looking into this further and we will provide an update here soon.

**Comment by tim-finnigan** [comment]
A new release just went out to fix this (2.4.26). I tested with the modified Dockerfile below and the installation succeeded:

```
FROM centos:7

WORKDIR /tmp

RUN yum -y install unzip
RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64-2.4.26.zip" -o "awscliv2.zip"
RUN unzip awscliv2.zip
RUN cd aws && ./install -u -i /tmp/aws-cli -b /tmp/aws-cli-bin
RUN /tmp/aws-cli-bin/aws --version
```

 Thanks again for reporting this, I’ll go ahead and close this as resolved.
